### PR TITLE
Create tod-cheat-sheet.json

### PR DESCRIPTION
**Pull request by manun1204**
Ia : https://duck.co/ia/view/total_overdose_cheat_sheet

### CODE REVIEW

**Comment by daxtheduck** [comment]
@manun1204 Hey!
        
Thanks for taking the time to contribute! We really appreciate it.

We work closely with every contributor to make Instant Answers the best they can be, so we appreciate your patience as we look over your code. From here, the process usually goes like this:

1. Pull Request is reviewed by the DuckDuckGo staff and community
2. Staff and community will leave feedback with any necessary updates to the function or design.
3. Once you've made any necessary corrections, then your Instant Answer will be merged and deployed live on DuckDuckGo!

If you have any questions along the way, feel free to ask them here. Our staff and community are also available on Slack to answer any questions you may have. If you'd like to join us there please send an email to <quackslack@duckduckgo.com>.

>More Info: https://duck.co/duckduckhack/submission_and_review

Thanks!

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

**Comment by edgesince84** [comment]
@manun1204,  thanks again for creating the TOD Cheat Sheet! You’ve been doing a really good job here and we'd like you to help make DuckDuckGo even better for yourself and others interested in gaming cheats.

When you're ready to move on, please: 

- Create a new Instant Answer for searches you do related to gaming cheats / codes

OR

- Test live <topic> Instant Answers (or those in development) for bugs and suggest improvements

Detailed instructions explaining how you can help are available at the top of this [page](https://github.com/duckduckgo/duckduckgo/wiki/Creating-or-Improving-Instant-Answers).

Thanks again!

File: share/goodie/cheat_sheets/json/totaloverdose.json [hunk 1]
```diff
@@ -0,0 +1,49 @@
+{
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/totaloverdose.json]: @manun1204 your ID here and the ID of the Instant Answer Page, as well as the filename was all match.

Your IA Page ID is `tod_cheat_sheet` so the ID here should be `tod_cheat_sheet`. Likewise, the filename should be `tod.json`

You can add `totaloverdose` as an `alias` so "totaloverdose cheat sheet" triggers this as well

Comment (moollaza) [inline share/goodie/cheat_sheets/json/totaloverdose.json]: For now i will change your IA Page ID, but you'll need to add `tod` as an alias

Comment (moollaza) [inline share/goodie/cheat_sheets/json/totaloverdose.json]: It looks like the game is spelled "Total Overdose".

I would suggest you change the name to `total_overdose_cheat_sheet` then "total overdose cheat sheet" will trigger this.

Comment (moollaza) [inline share/goodie/cheat_sheets/json/totaloverdose.json]: Sorry for the noise. I've updated the `id` of the IA page to `total_overdose_cheat_sheet`.

Please change the ID in the JSON to match that and change the filename to `total-overdose.json`.

**Comment by mintsoft** [comment]
@manun1204  I'm going to close this PR for now; feel free to reopen it whenever you like :+1:

**Pull request closed without merging**